.github/workflows/ci.yaml

```diff
@@ -29,3 +29,50 @@ jobs:
         run: |
           flit install -s
           pytest tests/unit
+
+  linux-integration_tests-sqlserver:
+    name: "Linux - integration tests - Python ${{ matrix.PYTHON_VERSION }} - SQL server with ${{ matrix.DRIVER }} driver"
+    runs-on: ubuntu-20.04
+    env:
+      CI: True
+    strategy:
+      fail-fast: false
+      matrix:
+        PYTHON_VERSION: ['3.8']
+        DRIVER: ["mssql"] #, "mssql-freetds"]
+    services:
+      DB:
+        image: mcr.microsoft.com/mssql/server:2019-latest
+        env:
+          ACCEPT_EULA: Y
+          SA_PASSWORD: pytsql-Test-123
+        ports:
+          - 1433:1433
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Fetch full git history
+        run: git fetch --prune --unshallow
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: ${{ matrix.PYTHON_VERSION }}
+          miniforge-variant: Mambaforge
+          miniforge-version: 4.11.0-2
+          use-mamba: true
+          environment-file: environment.yml
+          activate-environment: pytsql
+      - name: Install msodbcsql17 driver
+        shell: bash -l {0}
+        run: |
+          wget https://packages.microsoft.com/ubuntu/20.04/prod/pool/main/m/msodbcsql17/msodbcsql17_17.9.1.1-1_amd64.deb
+          ACCEPT_EULA=Y sudo apt install ./msodbcsql17_17.9.1.1-1_amd64.deb
+      - name: Wait for SQL Server

```

Comment:
Just curious: Do we still need this step after some other issues have been resolved?

Reply:
Probably not, but I think it's a good hygiene step. I don't know what's the level of assurance, but it might happen that by the time you want to run the tests the database is still not available.